README.md

```diff
@@ -39,6 +39,55 @@ library = foreman-ansible-modules/modules
 
 Now your playbooks and roles should have access to the modules contained in the repository for use, testing or development of new modules.
 
+## How to debug this repository
+
+Clone both ansible and this project
+
+```sh
+git clone https://github.com/ansible/ansible.git
```

Comment:
yeah, it could live in a `.gitignore`'d directory. In addition, this process will most likely be required for automated tests in the future.